de-lib/src/main/java/org/iplantc/de/client/services/impl/AnalysisServiceFacadeImpl.java

```diff
@@ -265,8 +265,8 @@ protected Void convertFrom(String object) {
     }
 
     @Override
-    public void stopAnalysis(Analysis analysis, DECallback<String> callback) {
-        String address = ANALYSES + "/" + analysis.getId() + "/stop";
+    public void stopAnalysis(Analysis analysis, DECallback<String> callback, String status) {
+        String address = ANALYSES + "/" + analysis.getId() + "/stop?job_status=" + status;
```

Comment:
I don't validate that it's a valid status here at all, but it's all hardcoded values for now, so I'm not sure if I need to?